save-plugins/warn-plugin/README.md

```diff
@@ -52,17 +57,22 @@ description = "My suite description"
 suiteName = "DocsCheck"
 
 [warn]
-execCmd="./detekt --build-upon-default-config -i"
-warningsOutputPattern = \w+ - \d+/\d+ - .*$
-output = stdout  # you can also use 'file' here to do fixes right into the test file (test files won't be broken or changed)
-# FixMe: what to do with a file configuration? Do we need to have an extra option with a path to this file?
-batchMode = false
-wanrningTextHasColumn = true
-wanrningTextHasLine = true
+execCmd = "./detekt --build-upon-default-config -i"
+warningsInputPattern = "// ;warn:(\\d+):(\\d+): (.*)"  # warning is set inside the comment in code, `//` marks comment start in Java
+warningsOutputPattern = "\\w+ - (\\d+)/(\\d+) - (.*)$"  # e.g. `WARN - 10/14 - Class name is in incorrect case`
+lineCaptureGroup = 1
+columnCaptureGroup = 2
+messageCaptureGroup = 3
+warningTextHasColumn = true
+warningTextHasLine = true
```

Comment:
need to add a property for EXACT match of warnings: exactWarningsMatch = true (default)
What if save will expect 5 warnings and receive 6 (if static analyzer is not configurable and can show only all warnings by the default)

Reply:
Or create an issue for this (this sounds better)

Reply:
#50